index.html

```diff
@@ -38,7 +39,7 @@ <h3>
                         </h3>
                     </div>
 
-                    <div class="card-block">
+                    <div class="card-body">
```

Comment:
`card-block` was not added to Bootstrap4 :
https://stackoverflow.com/questions/53750938/what-is-the-bootstrap-card-block-class